config/jobs/kubernetes/sig-k8s-infra/porche/sig-k8s-infra-porche-presubmits.yaml

```diff
@@ -0,0 +1,85 @@
+presubmits:
+  kubernetes-sigs/porche:
+  - name: pull-porche-build
+    decorate: true
```

Comment:
For some reason this one doesn't have a cluster annotation, in the oci-proxy prow files on which this is based..

https://github.com/kubernetes/test-infra/blob/4efb047a155a58c092b6716ba4e80641454cd1b0/config/jobs/kubernetes/sig-k8s-infra/oci-proxy/sig-k8s-infra-oci-proxy-presubmits.yaml#L72

Reply:
I forgot to do it before. Fix https://github.com/kubernetes/test-infra/pull/27925.